src/test/scala/PSO/ParticleSpec.scala

```diff
@@ -21,14 +21,14 @@ import org.scalatest.BeforeAndAfterAll
 import org.scalatest.WordSpecLike
 import org.scalatest.Matchers
 
-import com.trifectalabs.myriad.ProblemType
-import com.trifectalabs.myriad.pso.{ PSOConfiguration, PSOSystemFactory,
-  ParticleTopology, ReportRequest, Report, ComputeIteration }
-import com.typesafe.config.ConfigFactory
+import com.trifectalabs.myriad.{ParticleState, SwarmMaster, SampleFunctions,
+  ProblemType}
+import com.trifectalabs.myriad.pso._
 
-import akka.actor.ActorSystem
+import akka.actor.{Props, ActorSystem}
 import akka.testkit.{ DefaultTimeout, ImplicitSender, TestKit }
 import scala.concurrent.duration._
+import akka.testkit.TestActorRef
 
 class ParticleSpec
```

Comment:
Might just be me, but I'm not a huge fan of seeing random big numbers in the spec. I'd recommend that we check for function invocation primarily. If you disagree, maybe we can chat so I can get a better understanding of what these tests are actually doing.